pipeline.conf

```diff
@@ -358,40 +369,52 @@ filter {
-                "^(kvXtrct_)?usg$", "^(kvXtrct_)?ved$", "^(kvXtrct_)?sa$", "tbpm", "s,ql",".*sbmlretrieval\/allforms\?speciesid.*", "redlink",
-                "nsukey","^(kvXtrct_)?ll$", "maxadcountsperinterval", "method:#_memberaccess%\.url",
-                "infloat", "inajax", "ajaxmenu", "handlekey", "gf_page", "^(kvXtrct_)?fid$", "entryac",
-                "cssreloader2345", "cperpage", "classic", "chvzaa", "bustype", "beacon", "adcountintervalhours",
-                "^(kvXtrct_)?1$", "^(kvXtrct_)?4$", "^(kvXtrct_)?2$", "stichwort", "wx_fmt", "phpinfo", "agreed",
-                "^(kvXtrct_)?pg$", "arubalp", "arrs\d", "^(kvXtrct_)?highlight$", "imagefield\.(x|y)", "blast_rank",
-                "^(kvXtrct_)?rid$", "log\$", "route", "rqid", "subid", "affiliate", "justshown", "^anx.+",
-                "cfg_(basedir|imgtype|mediatype|not_allowall||softtype)", "^(kvXtrct_)?drv$", "^(kvXtrct_)?sz$", "^(kvXtrct_)?ti$",
-                "^(kvXtrct_)?(c|C)(7|9)$", "^(kvXtrct_)?oe$", "^(kvXtrct_)?channel$", "^(kvXtrct_)?gfe_rd$", "^(kvXtrct_)?1\'$", "^(kvXtrct_)?o$", "^(kvXtrct_)?oh$", "^(kvXtrct_)?efg$",
-                ".*\+{2,}.*", "^(kvXtrct_)?db\/cgi-bin\/instancebrowser\?db$", "^(kvXtrct_)?(json(p(-|_)?)?)?callback$", "^(kvXtrct_)?dir(_inc)?$",
-                "crazycache", "^(kvXtrct_)?cn$","fl", "^(kvXtrct_)?rev$", "kb_tick", "j24a75", "wb48617274", "x9wj", "^(kvXtrct_)?ei$", "^(kvXtrct_)?foo$",
-                "\'\/\*\*\/\'", "-d\+allow_url_include", "132\?cmd", "^(kvXtrct_)?;.+",
-                "fbclid", "nslookup_button", "^(kvXtrct_)?s?http:\/\/.*", "^(kvXtrct_)?ie$", "^(kvXtrct_)?iid$", "name\.\#.*",
-                "queryb64str", "qrahfyhjkjaasc6j", "$^(kvXtrct_)puid$", "patcher\?cmd", "sla6pddys", "yyue",
-                "^(kvXtrct_)?at[0-9]+$", ".*convert\(.*", ".*(\(|\)).*", "(kvXtrct_)?#$", ".*9,\+9,\+9", ".*9ehhdszt",
-                ".*%.*", ".*(x|y)_(mag|offset)", ".*sp\.nextform", "^(kvXtrct_)?skin$", "(kvXtrct_)?sg$", ".*redirect:\$\{\#a.*",
-                ".*R0BPnN9T.*", "^(kvXtrct_)?php.*", "^(kvXtrct_)?pollvars.*", ".*new(lang|page|remember|username).*", ".*amp;.*",
-                "^kvXtrct__escaped_fragment_", "^kvXtrct_'(a|A)", "^kvXtrct_.*;.*", "^kvXtrct_#.*", "^kvXtrct__.*", "^kvXtrct_,.*",
-                "^kvXtrct_an.*", "^kvXtrct_gt;.*", "kvXtrct_null", "^kvXtrct_page.*\/", "kvXtrct_x-nebulaxmlhttprequest",
-                "kvXtrct__t_t_t", "kvXtrct__dc", "kvXtrct____directive", "kvXtrct_amp\:.*", "kvXtrct_anx.*", "kvXtrct_",
-                "kvXtrct_\..*", "kvXtrct_--.*", "kvXtrct_acc(ept)?", "kvXtrct_\d+", "kvXtrct_.*\?.*", "kvXtrct_.*\/.*"
-                ]
+        # prune {
+        #     # a list of keys we want to prune. This list may grow over time.
+        #     blacklist_names => [".*(z|Z)(o|O|0){2}(m|M).*","(kvXtrct_)?xml_sitemap",
```

Comment:
Are these commented out to switch to an "allow" instead of a "deny" list or is there another reason?

Reply:
Really just here as a historic relic. I had originally tried using the prune plugin to remove unwanted keys. But the list of unwanted keys keeps growing, so it just became easier to allow certain keys. Of course, the list of allowed keys may grow over time if new URL parameters are added to existing Reactome components, but not all of them will need to be indexed in Elastic. I suppose if someone is interested in a key that's not in the allow-list, existing documents could be updated in-place (or re-indexed if the pipeline is updated though that's probably slower). The pipeline should be modified so that the new key is added to the allow-list so that future documents are created properly.

Reply:
I guess I can delete this stuff.

Reply:
Okay, if you want to delete it, I can approve the PR after that